03-Calling-an-API/src/components/NavBar.js

```diff
@@ -0,0 +1,174 @@
+                      className="btn-margin"
+                      onClick={handleLoginClick}
+                    >
+                      Log in
+                    </Button>
+                  </NavItem>
+                )}
+                {isAuthenticated && (
+                  <UncontrolledDropdown nav inNavbar>
+                    <DropdownToggle nav caret>
+                      <img
+                        src={profile.picture}
+                        alt="Profile"
+                        className="nav-user-profile"
+                      />
+                    </DropdownToggle>
+                    <DropdownMenu right>
+                      <DropdownItem header>{profile.name}</DropdownItem>
+                      <DropdownItem
+                        tag={RouterNavLink}
+                        to="/profile"
+                        className="dropdown-profile"
+                        activeClassName="router-link-exact-active"
+                      >
+                        <span className="icon icon-profile" /> Profile
+                      </DropdownItem>
+                      <DropdownItem
+                        id="qsLogoutBtn"
+                        onClick={handleLogoutClick}
+                      >
+                        <span className="icon icon-power" /> Log out
+                      </DropdownItem>
+                    </DropdownMenu>
+                  </UncontrolledDropdown>
+                )}
+              </Nav>
+              {!isAuthenticated && (
+                <Nav className="d-md-none" navbar>
+                  <NavItem>
+                    <Button
+                      id="qsLoginBtn"
+                      color="primary"
+                      block
+                      onClick={handleLoginClick}
+                    >
+                      Log in
+                    </Button>
+                  </NavItem>
+                </Nav>
+              )}
+              {isAuthenticated && (
+                <Nav className="d-md-none" navbar>
+                  <NavItem>
+                    <span className="user-info">
+                      <img
+                        src={profile.picture}
+                        alt="Profile"
+                        className="nav-user-profile d-inline-block"
+                      />
+                      <h6 className="d-inline-block">{profile.name}</h6>
+                    </span>
+                  </NavItem>
+                  <NavItem>
+                    <span className="icon icon-profile" />
+                    <RouterNavLink
+                      to="/profile"
+                      activeClassName="router-link-exact-active"
+                    >
+                      Profile
+                    </RouterNavLink>
+                  </NavItem>
+                  <NavItem>
+                    <span className="icon icon-power" />
+                    <a id="qsLogoutBtn" href="#" onClick={handleLogoutClick}>
```

Comment:
ESLint complains about this in the terminal when you start the project:

> The href attribute requires a valid value to be accessible.

Doesn't affect the running of the app but it would be good to eliminate any warnings that we can for the user.